sanic_jwt/__init__.py

```diff
@@ -1,57 +1,23 @@
 import logging
-logging.getLogger(__name__).addHandler(logging.NullHandler())
-
-from sanic.response import text
-
-from sanic_jwt.blueprint import bp as sanic_jwt_auth_bp
-from sanic_jwt.authentication import SanicJWTAuthentication
-
-from sanic_jwt import settings
-from sanic_jwt import exceptions
-from sanic_jwt import utils
-from sanic.views import HTTPMethodView
 
+from .authentication import Authentication
+from .initialization import Initialize
+from .initialization import initialize
+from .configuration import Configuration
+from .response import Response
+from .decorators import protected
+from .decorators import scoped
 
-def initialize(
-    app,
-    authenticate,
-    class_views=None,
-    store_refresh_token=None,
-    retrieve_refresh_token=None,
-    retrieve_user=None
-):
-    # Add settings
-    utils.load_settings(app, settings)
 
-    if class_views is not None:
-        for route, view in class_views:
-            if issubclass(view, HTTPMethodView) and isinstance(route, str):
-                sanic_jwt_auth_bp.add_route(
-                    view.as_view(),
-                    route,
-                    strict_slashes=app.config.SANIC_JWT_STRICT_SLASHES
-                )
-            else:
-                raise exceptions.InvalidClassViewsFormat()
-
-    # Add blueprint
-    app.blueprint(sanic_jwt_auth_bp, url_prefix=app.config.SANIC_JWT_URL_PREFIX)
-
-    # Setup authentication module
-    app.auth = SanicJWTAuthentication(app, authenticate)
-    if store_refresh_token:
-        setattr(app.auth, 'store_refresh_token', store_refresh_token)
-    if retrieve_refresh_token:
-        setattr(app.auth, 'retrieve_refresh_token', retrieve_refresh_token)
-    if retrieve_user:
-        setattr(app.auth, 'retrieve_user', retrieve_user)
+logging.getLogger(__name__).addHandler(logging.NullHandler())
```

Comment:
Are you planning on using the logger somewhere?

Reply:
Yeah. on the radar. I have some more ideas on this. but probably will take this out for v1.

Reply:
I was thinking in using it for loading certificates regarding #49

Reply:
I added some logging in d6ce8458dccad3d1f17c5e1dbb3dc82c79cbb607